### PR TITLE
🐛 Source [Hubspot, Zendesk-Support]: fix stream schemas

### PR DESCRIPTION
**Pull request by bazarnov**
## What
Based on these test reports:
- Hubspot: [test](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/nightly_builds/master/1697686413/a53a020c6abd51a6b718c1d4dc0d21dff7b423de/source-hubspot/1.5.1/output.html)
- Zendesk-Support: [test](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/nightly_builds/master/1697686413/a53a020c6abd51a6b718c1d4dc0d21dff7b423de/source-zendesk-support/2.0.0/output.html)

## How
- added missing fields:
   - hubspot: `aifeatures` to the `marketing emails` stream schema
   - zendesk-support: `reply_time_in_seconds` to the `ticket metrics` stream schema
- removed cross-dependency between `setup.py` and `airbyte-cdk` for `source-hubspot`
- updated `expected records` for both sources

## 🚨 User Impact 🚨
No impact is expected, the fields are added on top of the schema with no breaking changes.

### CODE REVIEW

**Comment by airbyte-oss-build-runner** [comment]
##  source-zendesk-support test report (commit [`a5ff64a6c3`](https://github.com/airbytehq/airbyte/pull/31606/commits/a5ff64a6c3398661cc3da0abacfee36562fbf50c)) - ✅

⏲️  Total pipeline duration: 08mn13s 

| Step                                                                   | Result   |
|:-----------------------------------------------------------------------|:---------|
| Build source-zendesk-support docker image for platform(s) linux/x86_64 | ✅       |
| Unit tests                                                             | ✅       |
| Acceptance tests                                                       | ✅       |
| Code format checks                                                     | ✅       |
| Validate metadata for source-zendesk-support                           | ✅       |
| Connector version semver check                                         | ✅       |
| Connector version increment check                                      | ✅       |
| QA checks                                                              | ✅       |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-hubspotzendesk-support_fix-schemas/1697715661/a5ff64a6c3398661cc3da0abacfee36562fbf50c/source-zendesk-support/2.0.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:a5ff64a6c3398661cc3da0abacfee36562fbf50c)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-hubspot test report (commit [`a5ff64a6c3`](https://github.com/airbytehq/airbyte/pull/31606/commits/a5ff64a6c3398661cc3da0abacfee36562fbf50c)) - ✅

⏲️  Total pipeline duration: 12mn11s 

| Step                                                           | Result   |
|:---------------------------------------------------------------|:---------|
| Build source-hubspot docker image for platform(s) linux/x86_64 | ✅       |
| Unit tests                                                     | ✅       |
| Integration tests                                              | ✅       |
| Acceptance tests                                               | ✅       |
| Code format checks                                             | ✅       |
| Validate metadata for source-hubspot                           | ✅       |
| Connector version semver check                                 | ✅       |
| Connector version increment check                              | ✅       |
| QA checks                                                      | ✅       |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-hubspotzendesk-support_fix-schemas/1697715661/a5ff64a6c3398661cc3da0abacfee36562fbf50c/source-hubspot/1.5.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:a5ff64a6c3398661cc3da0abacfee36562fbf50c)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

File: airbyte-integrations/connectors/source-hubspot/Dockerfile [hunk 1]
```diff
@@ -34,5 +34,5 @@ COPY source_hubspot ./source_hubspot
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=1.5.1
+LABEL io.airbyte.version=1.5.2
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-hubspot/Dockerfile:37]:
```suggestion
LABEL io.airbyte.version=1.6.0
```

File: airbyte-integrations/connectors/source-zendesk-support/Dockerfile [hunk 2]
```diff
@@ -25,5 +25,5 @@ COPY source_zendesk_support ./source_zendesk_support
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=2.0.0
+LABEL io.airbyte.version=2.0.1
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-zendesk-support/Dockerfile:28]:
```suggestion
LABEL io.airbyte.version=2.1.0
```

**Review by davydov-d — APPROVED** [review]

File: airbyte-integrations/connectors/source-hubspot/Dockerfile (hunk 1 above)
Comment (askarpets) [reply airbyte-integrations/connectors/source-hubspot/Dockerfile:37]: https://docs.airbyte.com/contributing-to-airbyte/resources/pull-requests-handbook/#examples

**Comment by airbyte-oss-build-runner** [comment]
##  source-zendesk-support test report (commit [`84de3cdc78`](https://github.com/airbytehq/airbyte/pull/31606/commits/84de3cdc78eca4b28823ccdbc69dbc3f6446d836)) - ❌

⏲️  Total pipeline duration: 07mn49s 

| Step                                                                   | Result   |
|:-----------------------------------------------------------------------|:---------|
| Build source-zendesk-support docker image for platform(s) linux/x86_64 | ✅       |
| Unit tests                                                             | ✅       |
| Acceptance tests                                                       | ✅       |
| Code format checks                                                     | ✅       |
| Validate metadata for source-zendesk-support                           | ✅       |
| Connector version semver check                                         | ✅       |
| Connector version increment check                                      | ✅       |
| QA checks                                                              | ❌       |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-hubspotzendesk-support_fix-schemas/1697794431/84de3cdc78eca4b28823ccdbc69dbc3f6446d836/source-zendesk-support/2.1.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:84de3cdc78eca4b28823ccdbc69dbc3f6446d836)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-zendesk-support test report (commit [`e0a8e745b1`](https://github.com/airbytehq/airbyte/pull/31606/commits/e0a8e745b15a3e53bab3076ad586fd27797b8070)) - ✅

⏲️  Total pipeline duration: 07mn29s 

| Step                                                                   | Result   |
|:-----------------------------------------------------------------------|:---------|
| Build source-zendesk-support docker image for platform(s) linux/x86_64 | ✅       |
| Unit tests                                                             | ✅       |
| Acceptance tests                                                       | ✅       |
| Code format checks                                                     | ✅       |
| Validate metadata for source-zendesk-support                           | ✅       |
| Connector version semver check                                         | ✅       |
| Connector version increment check                                      | ✅       |
| QA checks                                                              | ✅       |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-hubspotzendesk-support_fix-schemas/1697795144/e0a8e745b15a3e53bab3076ad586fd27797b8070/source-zendesk-support/2.1.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:e0a8e745b15a3e53bab3076ad586fd27797b8070)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-hubspot test report (commit [`e0a8e745b1`](https://github.com/airbytehq/airbyte/pull/31606/commits/e0a8e745b15a3e53bab3076ad586fd27797b8070)) - ✅

⏲️  Total pipeline duration: 13mn09s 

| Step                                                           | Result   |
|:---------------------------------------------------------------|:---------|
| Build source-hubspot docker image for platform(s) linux/x86_64 | ✅       |
| Unit tests                                                     | ✅       |
| Integration tests                                              | ✅       |
| Acceptance tests                                               | ✅       |
| Code format checks                                             | ✅       |
| Validate metadata for source-hubspot                           | ✅       |
| Connector version semver check                                 | ✅       |
| Connector version increment check                              | ✅       |
| QA checks                                                      | ✅       |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-hubspotzendesk-support_fix-schemas/1697795144/e0a8e745b15a3e53bab3076ad586fd27797b8070/source-hubspot/1.6.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:e0a8e745b15a3e53bab3076ad586fd27797b8070)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-hubspot test
```

**Pull request merged**